bin/select_inflation.R

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env Rscript
+# Tidyverse should already be included here
+library(tidyverse)
+library(reshape)
+library(cowplot)
+library(elbow)
+
+# Install the remaining necessary packages and then load
+#dir.create('./Rlibs', showWarnings = FALSE)
+#.libPaths('./Rlibs')
+
+#install.packages(c('reshape', 'cowplot', 'devtools'), 
+#  lib = './Rlibs', repos='https://ftp.osuosl.org/pub/cran/')
+#devtools::install_github("ahasverus/elbow", build_vignettes = FALSE, lib = './Rlibs')
+
+#library(reshape, lib = './Rlibs')
+#library(cowplot, lib = './Rlibs')
+#library(elbow, lib = './Rlibs')
+
+# Read in summaries of MCL Inflation parameter performance.
+fpaths <- list.files(pattern = 'summary.tsv')
+fs <- list()
+for(i in 1:length(fpaths)){
+  fs[[i]] <- read.delim(fpaths[[i]])
+}
+res <- do.call(rbind, fs)
```

Comment:
I think since these files are all just two lines each I'll stick thick this, but it's good to know! Would have to modify the docker image to install purrr otherwise.